packages/driver/src/cy/keyboard.ts

```diff
@@ -703,8 +705,9 @@ export class Keyboard {
             debug('skipping validation due to *skipCheckUntilIndex*', _skipCheckUntilIndex)
           }
 
-          if (key.simulatedDefaultOnly && key.simulatedDefault) {
-            key.simulatedDefault(activeEl, key, options)
+          // simulatedDefaultOnly keys will not send any events, and cannot be canceled
+          if (key.simulatedDefaultOnly) {
```

Comment:
This is my first time looking at this code. I was looking where this is used and I have to admit I'm still confused by the naming of this variable. What does `simulatedDefaultOnly` mean?? I think I understand the `simulatedDefault` part, but the `Only` means..?? It's doing absolutely nothing but the simulated default action?

Reply:
correct. this is used for special sequence `{selectall}` for example, since it represents ONLY an action, not any key presses